src/background/background.js

```diff
@@ -48,26 +63,49 @@ const sendToActiveTab = function (request, callback) {
 };
 
 browser.runtime.onMessage.addListener(async request => {
-	console.log('browser.runtime.onMessage', request);
+	if (request.getStorageType) {
+		return store.constructor.name;
+	}
+
+	if (request.login) {
+		return await store.initTalismanLogin(request.credential).catch(er => {
+			return er;
+		});
+	}
+
+	if (request.logOut) {
+		await store.logOut();
+	}
+
+	if (request.isAuthorized) {
+		const storeData = await store.isAuthorized();
+		return storeData ? { data: storeData } : false;
+	}
```

Comment:
Splitting StoreDevTools in two feels weird to me with all their requests mixed here. Maybe we should merge them back? Additionally I'd rename talisman related requests to include talisman in their names (e.g. talismanLogin, taismanLogout, getTalismanUser)